src/edu/stanford/dlss/was/WasapiDownloader.java

```diff
@@ -1,8 +1,38 @@
 package edu.stanford.dlss.was;
 
-public class WasapiDownloader
-{
-    public static void main(String[] args) {
-        System.out.println("I am WasapiDownloader. Hear me roar. Woo.");
+import java.io.*;
```

Comment:
i was OK with that because that's what i've been doing forever, but i don't think that's a particularly good justification, so i'm happy to be more limited in our imports if that's better practice.